config/reaction-oss/reaction-v3.1.0.mk

```diff
@@ -0,0 +1,34 @@
+###############################################################################
+### Reaction OSS v3.0.4
```

Comment:
The comment is out of sync. We could remove this comment to reduce the number of things that we need to worry about.